setup-machine-arktos.sh

```diff
@@ -182,6 +182,8 @@ pip3 install ipaddress
 pip3 install netaddr
 pip3 install kopf
 pip3 install PyYAML
+pip3 install netifaces
+pip3 install inquirer
```

Comment:
Remove both netifaces and inquirer. We won't be needing netifaces and inquirer as we only using environment variable to set interface name